ext/ffi/ffi.g

```diff
@@ -143,7 +158,7 @@ declaration_specifiers(zend_ffi_dcl *dcl):
 
 specifier_qualifier_list(zend_ffi_dcl *dcl):
 	"__extension__"?
-	(	?{sym != YY_ID || zend_ffi_is_typedef_name((const char*)yy_text, yy_pos - yy_text)}
+	(	?{sym != YY_ID || zend_ffi_is_typedef_name((const char*)yy_text, yy_pos - yy_text) || peek_sym() == YY__STAR}
```

Comment:
This should probably check for `YY__LBRACK` as well, to cater to `unknown_type[]`.